src/python/WMComponent/DBS3Buffer/MySQL/NewSubscription.py

```diff
@@ -40,15 +40,17 @@ def _createPhEDExSubBinds(self, datasetID, subscriptionInfo, custodialFlag):
 
         binds = []
         for site in sites:
-            binds.append({'id': datasetID,
-                          'site': site,
-                          'custodial': custodialFlag,
-                          'auto_approve': 1 if site in subscriptionInfo['AutoApproveSites'] else 0,
-                          'move': isMove,
-                          'priority': subscriptionInfo['Priority'],
-                          'phedex_group': phedex_group,
-                          'delete_blocks': delete_blocks,
-                          'dataset_lifetime': subscriptionInfo['DatasetLifetime']})
+            bind = {'id': datasetID,
+                    'site': site,
```

Comment:
can you protect site to satisfy to correct regexp. The schema only require it to be not null, so someone can pass "ABC" and it will be fine, but this is not what our sites are. Please put protection to validate this input value.

Reply:
This is already validated upstream, at the request spec level, e.g.:
https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/WMSpec/StdSpecs/StdBase.py#L1135